pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -38,6 +38,7 @@ assert x11Support -> tcl != null
 with stdenv.lib;
 
 let
+  buildPackages = pkgsBuildHost;
```

Comment:
Actually I wonder if we can get rid of this:

For example, `buildPackages.stdenv.cc` => `pkgsBuildBuild.targetPackages.stdenv.cc` which is kind of a mouthful, but I like it because it this is the `cc` for `BuildBuild` (holds for all `pkgsXxYy.targetPackages.stdenv.cc`, that is the `cc` for `XxYy`)

Reply:
I use `buildPackages` here because this is what people are most familiar with.